Std/Tactic/Omega/Frontend.lean

```diff
@@ -298,6 +298,9 @@ def pushNot (h P : Expr) : MetaM (Option Expr) := do
       return some (mkApp5 (.const ``Decidable.or_not_not_of_not_and []) P₁ P₂
         (.app (.const ``Classical.propDecidable []) P₁)
         (.app (.const ``Classical.propDecidable []) P₂) h)
+    | (``Not, #[P']) =>
+      return some (mkApp3 (.const ``Decidable.of_not_not []) P'
+        (.app (.const ``Classical.propDecidable []) P') h)
```

Comment:
this unnecessary `Classical` makes me vaguely grumble but I won't ask for any changes in this PR since there are other uses in the same function.

Reply:
Very happy to approve PRs removing `Classical` from `omega`!